apps/web/src/services/asset-inventory/collector/collector-history/CollectorHistoryPage.vue

```diff
@@ -239,6 +244,7 @@ watch(() => state.selectedStatus, (selectedStatus) => {
                              :page-size.sync="state.pageSize"
                              row-cursor-pointer
                              sortable
+                             :sort-by="state.sortBy"
```

Comment:
issue: The `state.sortDesc` is not currently bound. However, if you're not planning to provide a default value, wouldn't it be unnecessary to bind `state.sortDesc` to the table? Could `state.sortDesc` also be considered unnecessary in this case?

Reply:
Oh, I see! I'll remove it then!